### ISSUE TITLE
Bot does not start after config (Windows, Python 3.8)

### ISSUE THREAD

**Issue by izhzz** [instabot-py/instabot.py#2446]
Hi there, I need help with some technical stuff. Bot is not starting after config.

#### Select one:
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)

#### Environment
- [X] Are you hosting the bot on a VPS or other Cloud hosting?
```
Include more details about the environment
```

#### Operating System? (include version)
- [X] Windows Server 2016

#### Python Version Requirement
- [X] I am using Python 3.6 or above

#### Exact Python Version?
```
Python 3.8.0
```

#### Pip Version?
```
pip 19.3.1
```

#### Instabot Version?
```
INFO:instabot_py.config:Setting configuration from instabot.config.yml : OK


=============


Could not find user in settings. Would you like to add now? (yes/no):  : no
```

#### How you use the instabot
- [ ] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [x] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue

```
    ______Starting bot_____
Traceback (most recent call last):
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Scripts\instabot-py.exe\__main__.py", line 9, in <module>
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\instabot_py\__main__.py", line 394, in main
    bot = InstaBot(**configdict)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\instabot_py\instabot.py", line 60, in init
    self.persistence = PersistenceManager(config.get("database"))
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\instabot_py\persistence\manager.py", line 10, in new
    from instabot_py.persistence.sql import Persistence
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\instabot_py\persistence\sql.py", line 3, in <module>
    from sqlalchemy import Column, Integer, String, DateTime
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\sqlalchemy\__init__.py", line 8, in <module>
    from . import util as _util  # noqa
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\sqlalchemy\util\__init__.py", line 14, in <module>
    from ._collections import coerce_generator_arg  # noqa
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\sqlalchemy\util\_collections.py", line 16, in <module>
    from .compat import binary_types
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\sqlalchemy\util\compat.py", line 331, in <module>
    time_func = time.clock
AttributeError: module 'time' has no attribute 'clock'
```

#### Config

```
[xx]
username = xx
password = xxx
like_per_day = 198
comments_per_day = 0
max_like_for_one_tag = 36
follow_per_day = 260
follow_time = 36026
unfollow_per_day = 247
tag_list = ["gym", "fitness", "fitnesslifestyle"]
unfollow_selebgram = false
unfollow_probably_fake = true
unfollow_inactive = true
unfollow_recent_feed = false

```

**Comment by next007apple** [comment]
Fix Problem:
Install Python 3.7

Note:
In Python 3.8 have issue in time.clock()

**Comment by izhzz** [comment]
> 
> 
> Fix Problem:
> Install Python 3.7
> 
> Note:
> In Python 3.8 have issue in time.clock()

Thanks. After downgrading to 3.7 following error occurs:
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "C:\Users\Administrator\Downloads\new pypyp\instabot.py-master\instabot_py\__init__.py", line 1, in <module>
    from .instabot import InstaBot
  File "C:\Users\Administrator\Downloads\new pypyp\instabot.py-master\instabot_py\instabot.py", line 17, in <module>
    import instaloader
ModuleNotFoundError: No module named 'instaloader'

python -m instabot_py -c sample.instabot.config.yml

**Comment by marcelofox** [comment]
Did you find solution for this error?

**Comment by meestaShin** [comment]
developers have stopped this project for a long time. They announced it.

**Comment by nickgryg** [comment]
@marcelofox I've updated instabot-py in PyPi, you can try the latest version 0.5.12:
```python3 -m pip install instabot-py --no-cache-dir --upgrade```

**Comment by nickgryg** [comment]
This issue has been fixed in PR #2468
